### ISSUE TITLE
[CAIA a11y Research - PILOT]: IIR, Welcome Pathway (Mobile) (Nov. 2024)

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#95524]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATE:**  12/4/2024 at 1 p.m. 
> - **TESTING:** Mobile: iPhone - VoiceOver
> - **TYPES OF ASSISTIVE TECH:**  Desktop Screen Reader, Mobile Screen reader, Magnification/Zoom, Speech input Tech (Sire, Dragon) 
> - **TESTING ENVIRONMENT:** [Codespace] (🔗 `Pending Dev.`)
> - **RESEARCH MATERIALS:** [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20research%20plan.md), [Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20conversation%20guide.md) & [Research Goals](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/research%20plan%202024.md#research-goals)

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- **Pilot**: @rileyorr 
- **Co-pilot**: @sara-amanda 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: IIR (Iterate, Innovate, Run)
- OCTO-DE product owner: Jeff Barns
- Product manager: Pete Egan
- Designer: Liza McRuer/Raquel Eisele
- FE engineer: John Rodriguez/Nick Sayre
- Product/team Slack channel: #iir-product-teams-public

</details>

## Research Goals

<details><summary>Toggle to view team research goals</summary>

`Not yet provided`

For now ... please refer to "About the Research" section in the research ticket #94733 

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:**  Week of 11/25/2024
- **Prototype**: (🔗 `Pending`) 
- [Research Plan] (🔗 `Pending`) 
- [Research Convo Guide] (🔗 `Pending`) 
</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: @mutiaadhocteam will test using `AT` on `Browser` and `Device`
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @NaomiPMC  in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11ys**
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] HOLD placed the week of 11/11/2024 on CAIA Calendar
- [x] Hold updated for week of 11/25/2024
- [x] **Determine date and time for pilot** `12/4/2024` Scheduled 12/3/2024
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Receive Research Goals** add to ticket section above for goals.
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-onboarding/assistive-tech%20research/AT%20conversation%20guide.md)
- [x] Request AT device and tech to be used in sessions Mobile - iPhone VoiceOver
- [x] **Review prototype** in `Codespace`
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: Note Taking:** [Notes doc](https://docs.google.com/document/d/1zE737Vj5pk024n0jLbRuHiHZ8EhGNwjd0MrtqlolnUE/edit?tab=t.0#heading=h.7b53y4dqihzk)
- [x] Share findings with team from Pilot doc.
```

**Comment by sara-amanda** [comment]
## Timeframe Shift 
- 2 weeks until pilots
- Week of November 25 for pilots (TBD)

### Slack Update
`iir-product-teams-public`

> Raquel Eisele [Yesterday at 5:28 PM](https://dsva.slack.com/archives/C05RJS5DANT/p1731450522330639?thread_ts=1731425468.647049)
> Hey Jamie, I just had a conversation with my dev and it sounds like sadly we are not ready to conduct the pilot this week. If we implement the changes you highlighted we believe we can be ready for our pilot in two weeks and if we don't we believe we will be ready by next week. Can we adjust our plans with your team to accommodate that change?

cc: @coforma-jamie

**Comment by sara-amanda** [comment]
## Pilot Scheduled

- **Date**:  12/4/2024 at 1 p.m. 
- **Pilot**: @rileyorr 
- **Co-Pilot**: @sara-amanda 
- **Device**: iPhone
- **AT**: VoiceOver
- **Browser**: TBD
- **Prototype**: `Staging` 
 
### Prototype
The following URLS were grabbed from CC Ticket 82257, please advise if we should use different URLs. 
- https://staging.va.gov/my-va/welcome-va-setup/review-information/contact-information
- https://staging.va.gov/my-va/welcome-va-setup/review-information/contact-information/edit-mobile-phone
- https://staging.va.gov/my-va/welcome-va-setup/review-information/contact-information/edit-email-address
- https://staging.va.gov/my-va/welcome-va-setup/review-information/contact-information/edit-mailing-address
- https://staging.va.gov/my-va/welcome-va-setup/review-information/confirmation

#### User Log In 
- When a net new LOA1 user logs in they will be greeted with a Welcome to MyVA modal that gives them an idea of what they can expect from MyVA. This can be dismissed and will not show again due to the 'user creation date' at initial sign in being tracked under Profile.
- When an authenticated LOA3 user lands on MyVA they will be prompted with the 'You have missing contact info' (if applicable) and the link to the form where they can edit their mobile phone, email address, and mailing address.

**Ref. CC Tix:** #82257

cc: @coforma-jamie @raquelou

**Comment by rileyorr** [comment]
@sara-amanda I updated your comment to list @mutiaadhocteam as the CAIA team member conducting the pilot. I listed myself as co-pilot alongside you if you still wanted to attend

**Comment by rileyorr** [comment]
Session is completed. @mutiaadhocteam I tagged you in a couple comments in [the notes doc](https://docs.google.com/document/d/1zE737Vj5pk024n0jLbRuHiHZ8EhGNwjd0MrtqlolnUE/edit?usp=sharing) to makes sure I fully captured your experience prior to sending feedback to the team. 

Some feedback I noted: 
- **iPhone bug discovered**
   - When submitting updates to the phone number and email address, the focus is being sent to the bottom of the page either to the ‘Feedback’ button or another link in the footer
   - Riley tested this on Android and the focus was working as expected by going to the success alert upon submitting changes to the phone number and email address 

- **Recommendation: Remove the ‘Close’ X button on the welcome modal** 
   - Having two buttons that close the modal (the 'X' and 'Continue' buttons) is confusing
   - Maria recommends that the focus go to the modal’s header instead so it just takes 1 swipe to navigate to the ‘Continue’ button 

- **Android bug discovered after session** 
   - When a mobile phone number is added it does not let me swipe to it
   - I can swipe to ‘Mobile phone number heading 2’ and then ‘Edit mobile phone number’ but I cannot swipe to the number itself. The number is never read out to the screen reader. 

cc @sara-amanda

**Comment by sara-amanda** [comment]
> [!NOTE]
> ## CAIA INTERNAL
> - [x] This work has been completed
> - [x] Closing ticket
> - [x] Adding to tracker for CAIA Sprint 140
